CHANGELOG.md

```diff
@@ -15,8 +15,11 @@
 
 **Fixed bugs:**
 
-- EC2 autodetection is broken [\#691](https://github.com/chef/ohai/issues/691)
+- KVM modules is a bogus way to detect a KVM host [\#749](https://github.com/chef/ohai/issues/749)
```

Comment:
This KVM issue and the issue below it were closed, but I failed to stick the exclude from changelog label on them. They were closed out without code changes so they really shouldn't be on here. I've tossed the labels on, but the changelog should probably get updated